mslib/msui/sideview.py

```diff
@@ -44,11 +44,45 @@
 WMS = 0
 
 
+class CustomColorDialog(QtWidgets.QColorDialog):
```

Comment:
that is now doubling the code lines, 

there shoukd be only one definition, you can move it to 
mpl_qtwidget and import it then.